packages/@guidesmiths/cuckoojs-schematics/src/systemic-scaffolding/files/opentelemetry.js

```diff
@@ -0,0 +1,20 @@
+const { NodeTracerProvider } = require('@opentelemetry/sdk-trace-node');
```

Comment:
It is not clear to me whether it is better to use this module or `[@opentelemetry/auto-instrumentations-node](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/metapackages/auto-instrumentations-node#readme)`. I think the latter is more configurable, but it also includes the most common instrumentation, which increases the weight.

There's another module [`@opentelemetry/sdk-node`](https://www.npmjs.com/package/@opentelemetry/sdk-node), which IMHO is more readable

Reply:
`auto-instrumentations-node` seems to be a package that gathers several instrumentations, which users may not need by default. You still have to resort to `sdk-trace-node`.

`@opentelemetry/sdk-node` looks like a package to do more stuff, like using exporters, and simplify the setup. It is in beta though. I'm not sure if we would want to include exporters by default. My initial idea was to add basic instrumentations, which it's something the user may want to have by default. I'm not sure if they would also want to use exporters, and if it would imply to make certain assumptions which could lead to make the template less agnostic.

Happy to discuss it and choose the best options 👍